qa/tasks/mgr/dashboard/test_cluster.py

```diff
@@ -0,0 +1,17 @@
+from .helper import DashboardTestCase
+
+
+class ClusterTest(DashboardTestCase):
+    AUTO_AUTHENTICATE = False
+
+    def setUp(self):
+        super(ClusterTest, self).setUp()
+        self.reset_session()
+
+    def test_get_status(self):
+        self._get('/api/cluster/status')
+        self.assertStatus(200)
+
+    def test_update_status(self):
+        self._put('/api/cluster/update_status', {'cluster_status': 'POST_INSTALLED'})
+        self.assertStatus(200)
```

Comment:
We need a test that checks that the stored status is later retrieved.

Reply:
Done.